app/views/auth/registrations/new.html.haml

```diff
@@ -53,6 +52,14 @@
               required: false,
               wrapper: :with_label
 
+  - if Setting.min_age.present?
+    .fields-group
+      = f.input :date_of_birth,
+                as: :date_of_birth,
+                hint: t('simple_form.hints.user.date_of_birth', age: Setting.min_age.to_i),
```

Comment:
This string needs plural forms.

Reply:
https://github.com/mastodon/mastodon/pull/34636